backend/corpora/dataset_processing/process.py

```diff
@@ -9,12 +9,30 @@
 import numpy
 import scanpy
 
+try:
+    from ..common.entities.dataset import Dataset
+    from ..common.corpora_orm import DatasetArtifactFileType, DatasetArtifactType
+
+except ImportError:
+    # We're in the container
+    sys.path.append("/code")
+    from common.entities.dataset import Dataset
+    from ..common.corpora_orm import DatasetArtifactFileType, DatasetArtifactType
+
+# This is unfortunate, but this information doesn't appear to live anywhere
+# accessible to the uploader
+DEPLOYMENT_STAGE_TO_URL = {
```

Comment:
Can we put this in the corpora/${stage}/config

Reply:
or it could be passed to the container by the step function.

Reply:
but this works for now.

Reply:
Yeah I think I would be better to to live in some deployment config or terraform and get passed to the container.